### PR TITLE
 Add metrics for secrets counts 

### PR DESCRIPTION
**Pull request by mcornaton**

### CODE REVIEW

File: src/main/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretService.java [hunk 1]
```diff
@@ -17,22 +17,36 @@
...
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
 
+import javax.annotation.PostConstruct;
 import java.util.Optional;
 
 @Slf4j
 @Service
 public class TeeTaskComputeSecretService {
+    private static final String METRICS_PREFIX = "iexec.sms.secrets.teeTaskCompute.";
```
Comment (jbern0rd) [inline src/main/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretService.java:31]: Did you check how this is rendered in prometheus metfics names ?

[the remark this answers is not in the document]
Comment (jbern0rd) [reply src/main/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretService.java:31]: I did a check and am not fond of `iexec_sms_secrets_teeTaskCompute_initial_total`

File: src/main/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretService.java [hunk 2]
```diff
@@ -17,22 +17,36 @@
...
     private final EncryptionService encryptionService;
+    private final Counter addedSecretsSinceStart = Metrics.counter(METRICS_PREFIX + "added");
 
     protected TeeTaskComputeSecretService(
             TeeTaskComputeSecretRepository teeTaskComputeSecretRepository,
             EncryptionService encryptionService) {
         this.teeTaskComputeSecretRepository = teeTaskComputeSecretRepository;
         this.encryptionService = encryptionService;
+
+        Metrics.gauge(METRICS_PREFIX + "stored", this, teeTaskComputeSecretService -> teeTaskComputeSecretRepository.count());
```
[the remark this answers is not in the document]
Comment (mcornaton) [reply src/main/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretService.java]: Thanks for the idea! It's easier this way. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/181/commits/98c8ab71a0716277dff1314c4078c13e11000334.

File: src/main/java/com/iexec/sms/secret/web2/Web2SecretService.java [hunk 3]
```diff
@@ -20,21 +20,36 @@
...
     private final Web2SecretRepository web2SecretRepository;
 
+    private final Counter addedSecretsSinceStart = Metrics.counter(METRICS_PREFIX + "added");
+
     protected Web2SecretService(EncryptionService encryptionService,
                                 Web2SecretRepository web2SecretRepository) {
         super(encryptionService);
         this.web2SecretRepository = web2SecretRepository;
+
+        Metrics.gauge(METRICS_PREFIX + "stored", this, web2SecretService -> web2SecretRepository.count());
```
[the remark this answers is not in the document]
Comment (mcornaton) [reply src/main/java/com/iexec/sms/secret/web2/Web2SecretService.java]: Thanks for the idea! It's easier this way. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/181/commits/98c8ab71a0716277dff1314c4078c13e11000334.

File: src/main/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretService.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (mcornaton) [reply src/main/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretService.java]: Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/181/commits/197105af8e360914f388cf0a70934bb33c96596f.

**Review by jbern0rd — APPROVED** [review]
Thanks !

**Pull request merged**